scripts/check-theorem-list

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+# check-theorem-list - check if theorem list output is okay
```

Comment:
"_is okay_" is a bit vague and does not give much additional information vs. "check theorem list".

Maybe "_check when that generating the theorem list, `metamath.exe` does not encounter any error or warning, including missing bibliographic references_" would be clearer.

Reply:
Changed.